webui/react/src/components/ResourcePoolCard.stories.tsx

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
 
+import { getResourcePools } from 'services/api';
+
 import ResourcePoolCard from './ResourcePoolCard';
 
+const resourcePools = getResourcePools();
+
 export default {
   component: ResourcePoolCard,
   title: 'ResourcePoolCard',
 };
 
 export const Default = (): React.ReactNode => {
-  return <ResourcePoolCard containerStates={[]} rpIndex={0} />;
+  return <ResourcePoolCard
+    containerStates={[]}
+    resourcePool={resourcePools[Math.floor(Math.random()*length)]} />;
```

Comment:
**question:** is `length` defined?

Reply:
it is defined in browser that's by TS didn't complain. i fixed it